docs/testing.md

```diff
@@ -0,0 +1,148 @@
+# Testing Using Docker
+
+**Table of Contents**
+
+- [Installation](#Installation)
+  - [Docker configuration](#docker-configuration)
+  - [Python configuration](#python-configuration)
+- [Basic Docker commands](#basic-docker-commands)    
+
+
+## Installation
+
+#### Docker configuration
+
+The base Docker installation methods for popular Linux distributions are listed here:
+	https://docs.docker.com/engine/install/
+
+Docker is not supported by **Fedora 33** officially. On my host system I used the following method (Moby based approach):
```

Comment:
please separate that into a sub-part with header like `Docker on Fedora ...`

Reply:
Done